backend/mora/service/employee.py

```diff
@@ -35,6 +35,200 @@
                             url_prefix='/service')
 
 
+class BrugerRequestHandler(handlers.RequestHandler):
+    __slots__ = ['details_requests']
+    role_type = "employee"
+
+    def prepare_create(self, req):
+
+        """Create a new employee
```

Comment:
Although it probably out to be, this isn't actually where we get the documentation. Rather, it's extracted from `detail_writing` — which you also provide. Any docstring here should be more about the code. At least for now :/

(Note that the other classes don't actually provide one.)

Reply:
I had skipped this exercise - I only had the detail_writing-edit part documented.
I have now moved the create-docu there too and outfitted it with a "type": "employee"